Foundation/CGFloat.swift

```diff
@@ -615,174 +619,178 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
 // tgmath
 //===----------------------------------------------------------------------===//
 
+#if os(Android)
+import Glibc
+#endif
+
 @_transparent
 public func acos(_ x: CGFloat) -> CGFloat {
-    return CGFloat(acos(x.native))
+    return CGFloat(acos(x.double))
```

Comment:
Noted & it’s not @_transparent. I tried making it internal and got a compier assertion:
SIL verification failed: function_ref inside fragile function cannot reference a private or hidden symbol: (SingleFunction && RefF->isExternalDeclaration()) || RefF->hasValidLinkageForFragileRef()